src/pages/experiences/experiences-component.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import cx from 'classnames';
+import loadable from '@loadable/component'
+import styles from './experiences-styles.module';
+
+const ExpertMode = loadable(() => import('./data-globe'));
+const FeaturedMode = loadable(() => import('./featured-globe'));
```

Comment:
For the loadable pattern to work we need to add a condition on the `render` of the `loadable component`. The way I was envisioning  was to add a watcher to the `onLoad` function of both `Globes` that would add a param to the URL whenever the first `Globe` was rendered on the screen so only then we would be `mounting` the second `Globe` and hence start the `fetch` of its dependencies.
Imagine that scenario for the `data-globe`:

Set some initial state for the mount params

`data-globe-initial-state.js`

```js
export default {
// ... more initial state attributes 
  globe: {
    mountFeaturedGlobe: false,
    mountExpertGlobe: true
  }
}
```

Update the param the first time the globe renders on the screen

`data-globe.js`

```js
const handleMapLoad = (view, setDataGlobeSettings) => {
 // .. more stuff to do on load
  loadModules(["esri/core/watchUtils"]).then(([watchUtils]) => {
    watchUtils.whenNotOnce(view, 'updating', () => { setDataGlobeSettings({ mountFeaturedGlobe: true });})
  })
}
```

Check whenever the URL param changes and load the component when it is set to `true`

`experiences-component.jsx`

```jsx
<>
  <div className={cx({ [styles.hidden]: !isExpertMode })}>
    {mountExpertGlobe && <ExpertMode handleSwitch={() => { preserveParams(switchToFeature) }} />}
  </div>
  <div className={cx({ [styles.hidden]: isExpertMode })}>
    {mountFeaturedGlobe && <FeaturedMode handleSwitch={() => { preserveParams(switchToExpert) }} />}
  </div>
</>
```